### ISSUE TITLE
[Bug - models] Inconsistent Model Replacement Results

### ISSUE THREAD

**Issue by NiveStinks** [Traben-0/Entity_Model_Features#58]
**Describe the bug**
A custom zombie model I made is not replacing the vanilla model properly. It _has_ worked, but does so very inconsistently at about 10% of the time I've loaded the game since building my resource pack. The purpose of the model was to make it so the zombie's limbs were not mirrored in order to augment the custom texture I made. In-game, however, the texture is applied but the model is not, which causes the limbs to remain mirrored. This is despite the fact that the texture is in the cem folder along with the zombie.jem.

**Expected behavior**
This is how the zombie should look:
![image](https://github.com/Traben-0/Entity_Model_Features/assets/84201620/65c391c3-c610-4141-b3a9-362721d60f7e)

**Screenshots**
This is how the zombie looks in-game the majority of the time:
![image](https://github.com/Traben-0/Entity_Model_Features/assets/84201620/e745fe0c-3cfd-4dcc-9518-1a22d539f4e1)
As you can see, the legs and arms are mirrored rather than distinct.

**(please complete the following information):**
 - Minecraft Version: 1.19.2
 - EMF version 0.2.9
 - ETF version 4.4.4 (I do have ETF installed, but I'm not using it in tandem with EMF in this instance)

**Additional context**
The model was made in blockbench and the resourcepack is structured properly. I've re-exported the model a few times in an attempt to fix it, and I've closed and relaunched the game as well, in tandem with disabling and re-enabling the mod. None of these methods have worked consistently. I'm hoping this is a really dumb mistake or misunderstanding on my part.

Here is the optifine/cem folder containing zombie.jem and zombie.png
[optifine.zip](https://github.com/Traben-0/Entity_Model_Features/files/11687472/optifine.zip)

**Comment by Traben-0** [comment]
does it work every time if only etf and emf are installed?

**Comment by NiveStinks** [comment]
Yes, I do have plenty of other mods, but nothing that should impact zombies or how they're rendered. I'll make a separate instance with only etf, emf, and my resourcepack and see if it changes anything.

**Comment by Traben-0** [comment]
there is a known bug where some mods affect the model loading order, is why i ask

**Comment by NiveStinks** [comment]
Do you have a list of those mods handy? Would my modlist help at all?

**Comment by Traben-0** [comment]
not a difinitive list, and not on me atm

**Comment by NiveStinks** [comment]
Okay, it's most likely a compat issue of some sort, I just did what you asked and the zombie model worked on the first attempt which has never happened before.
